Frameworks/UIKit.Xaml/Label.xaml.cpp

```diff
@@ -0,0 +1,81 @@
+﻿//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+// clang-format does not like C++/CX
+// clang-format off
+#include "pch.h"
+#include "Label.xaml.h"
+
+#include "ObjCXamlControls.h"
+
+using namespace Microsoft::WRL;
+using namespace Platform;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+
+namespace UIKit {
+
+Label::Label() {
+    InitializeComponent();
+    Name = L"UILabel";
```

Comment:
I'll move it up to UILabel.mm so we can set the name of the actual (potentially derived) class.

---
In reply to: [87285187](https://github.com/Microsoft/WinObjC/pull/1318#discussion_r87285187) [](ancestors = 87285187)